### PR TITLE
feat/design: explore screen and other screens

### PR DESCRIPTION
**Pull request by bokuhe**
explore screen feature
- fetch recommended channel
- add skeleton view (loading)
- apply design

design
- user profile wallet
- reformatting wallet creation screen text
- settings screen

added features
+ remove List NFT feature
+ reconf authentication screen

### CODE REVIEW

File: src/palm-react/hooks/firestore/useFsChannels.ts [hunk 1]
```diff
@@ -10,12 +13,45 @@ export type UseFsChannelsReturn = {
...
+    } catch (e) {
+      setRecommendedChannelUrls([])
+      console.error(e)
+    }
+  }
+
+  useEffect(() => {
+    fetchRecommendedChatUrls()
+    setChannels([])
+  }, [])
```
Comment (denniswon) [inline src/palm-react/hooks/firestore/useFsChannels.ts:46]: just put this in useExploreChannels hook

**Review by denniswon — CHANGES_REQUESTED** [review]

File: src/palm-react-native-ui-kit/components/ProfileHeader.tsx [hunk 2]
```diff
@@ -143,7 +143,7 @@ const ProfileHeader = React.memo(
             <ProfileHeaderChatButton userProfileId={userProfileId} />
           )}
 
-          <ProfileWalletAddress userAddress={userAddress} />
+          {!isMyPage && <ProfileWalletAddress userAddress={userAddress} />}
```
[the remark this answers is not in the document]
Comment (bokuhe) [reply src/palm-react-native-ui-kit/components/ProfileHeader.tsx:147]: It is not displayed only on mypage. The wallet display method for mypage and other user profile pages is different.

File: src/palm-core/consts/url.ts [hunk 3]
```diff
@@ -0,0 +1,9 @@
+export const PALM_HOMEPAGE_URL = 'https://palmapp.xyz'
+export const PALM_TERMS_OF_SERVICE_URL = 'https://palmapp.xyz/legal/tos'
+export const PALM_PRIVACY_POLICY_URL = 'https://palmapp.xyz/legal/privacy'
+export const ISSUE_REPORTING_URL = 'https://palm.canny.io/issue-reporting'
+
+export const WAITLIST_URL =
+  'https://raw.githubusercontent.com/palmapp-xyz/assets/main/mobile/alpha/config_v2.json?cache=0'
+export const RECOMMENDED_CHANNELS_URL =
+  'https://raw.githubusercontent.com/palmapp-xyz/assets/main/mobile/recommended-chat.json?cache=0'
```
[the remark this answers is not in the document]
Comment (bokuhe) [reply src/palm-core/consts/url.ts:9]: It is a static url that does not change depending on the environment, so is it necessary to put it in .env?

File: src/palm-react-native-ui-kit/components/atoms/SkeletonView.tsx [hunk 4]
```diff
@@ -0,0 +1,46 @@
+import { COLOR } from 'palm-core/consts'
+import React, { ReactElement } from 'react'
+import { Animated, Easing } from 'react-native'
+
+const SkeletonView = (props: {
```
[the remark this answers is not in the document]
Comment (bokuhe) [reply src/palm-react-native-ui-kit/components/atoms/SkeletonView.tsx:5]: The term Skeleton is widely used in front-end development to refer to a skeleton-like UI that is displayed during loading.
I'll add comments with explanations to SkeletonView.

File: src/palm-react/hooks/firestore/useFsChannels.ts (hunk 1 above)
Comment (bokuhe) [reply src/palm-react/hooks/firestore/useFsChannels.ts]: changed

File: src/palm-react-native-ui-kit/components/ProfileHeader.tsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (lwj010) [reply src/palm-react-native-ui-kit/components/ProfileHeader.tsx:147]: I think it's okay? If it's other users profile then show only user address and if not show token balances also

File: src/palm-core/consts/url.ts (hunk 3 above)
[the remark this answers is not in the document]
Comment (denniswon) [reply src/palm-core/consts/url.ts:9]: I think for privacy of our urls

**Review by denniswon — APPROVED** [review]

**Pull request merged**